src/theme/QueryTable/QueryTable.js

```diff
@@ -36,18 +36,18 @@ export const QueryTable = ({
     addUpdate();
   };
 
-  const subHeader = hasUpdate ? (
+  const subHeader = !!hasUpdate && (
```

Comment:
the `hasUpdate` name sounds boolean. What is the value? I'm just curious to know why do you need the double negative.

Reply:
I did it to be double safe its a boolean at evaluating the `&&` operation